RadixWallet/Features/SettingsFeature/Settings+Reducer.swift

```diff
@@ -15,8 +21,10 @@ public struct Settings: Sendable, FeatureReducer {
 
 		public var shouldShowMigrateOlympiaButton: Bool = false
 		public var userHasNoP2PLinks: Bool? = nil
-
-		public init() {}
+		public var deepLinkToDisplayMnemonics: Bool
```

Comment:
uuuuuuuhmm 😩😩😩😩😩 this is terrible... but I cant come up with anything better :/ ideas?